tests/jerry/es2015/proxy_is_extensible.js

```diff
@@ -12,8 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// TODO: Update these tests when the internal routine has been implemented
```

Comment:
It would be good to add a few extra tests like this:
```js
var target = {};

var h1 = {
    isExtensible(target) {
        print('h1 ext')
        return true;
    },
}

var h2 = {
    isExtensible(target) {
        print('h2 ext')
        return false;
    },
}

var p1 = new Proxy(target, h1);
var p2 = new Proxy(target, h2);

print(Object.isExtensible(p1));     // true
try {
    print(Object.isExtensible(p2));
} catch(ex) {
    print("expected typeerror ok");
}

var p3 = new Proxy(p1, h1);
var p4 = new Proxy(p1, h2);

print(Object.isExtensible(p3));     // true

try {
    print(Object.isExtensible(p4));
} catch(ex) {
    print("expected typeerror ok");
}

var p5 = new Proxy(p2, h2);

try {
    print(Object.isExtensible(p5));
} catch(ex) {
    print("expected typeerror ok");
}

Object.preventExtensions(target);
print(Object.isExtensible(p5));     // false

```

Reply:
I've been updated the test file, can you check it if it is good now?